src/GraphQL/Resolver.hs

```diff
@@ -134,17 +133,24 @@ data Result a = Result [ResolverError] a deriving (Show, Functor, Eq)
 aggregateResults :: [Result Value] -> Result Value
 aggregateResults r = toValue <$> sequenceA r
 
+throwE :: Applicative f => ResolverError -> f (Result Value)
```

Comment:
We already do this in Validation.hs. Happy to rename, but totally bored of rewriting the same `Result` expression a dozen times over.